recipes/pcapplusplus/cmake/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+find_package(pcapplusplus REQUIRED CONFIG)
```

Comment:
```suggestion
find_package(PcapPlusPlus REQUIRED CONFIG)
```

Reply:
Fixed in https://github.com/conan-io/conan-center-index/pull/20296/commits/9f304cbee9a2a61149277c00d741bb16ec521467